netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -279,23 +279,6 @@ public NettyChannelBuilder maxInboundMetadataSize(int bytes) {
     return this;
   }
 
-  /**
-   * Equivalent to using {@link #negotiationType(NegotiationType)} with {@code PLAINTEXT} or
-   * {@code PLAINTEXT_UPGRADE}.
-   *
-   * @deprecated use {@link #usePlaintext()} instead.
-   */
-  @Override
-  @Deprecated
-  public NettyChannelBuilder usePlaintext(boolean skipNegotiation) {
-    if (skipNegotiation) {
-      negotiationType(NegotiationType.PLAINTEXT);
-    } else {
-      negotiationType(NegotiationType.PLAINTEXT_UPGRADE);
```

Comment:
I'd _really_ like to kill NegotiationType. We may want to introduce a new method for this before deleting it.